Pipfile

```diff
@@ -4,14 +4,14 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-flask = "==2.3.3"
-flask-githubapp = "*"
+flask = "==3.0.3"
```

Comment:
🎉 we can now upgrade flask

History here is v3 broke compatibility with `flask-githubapp` and we had to revert due to runtime failures:
https://github.com/advanced-security/ghas-reviewer-app/pull/24
https://github.com/advanced-security/ghas-reviewer-app/pull/25